ports/jumpoffthebridge/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "jumpoffthebridge.zip",
+  "items": [
+    "Jump Off The Bridge.sh",
+    "jumpoffthebridge"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Jump Off The Bridge",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "Jump Off The Bridge is a platformer with an emphasis on difficulty and unique level design. Each level, the player is required to jump off the bridge, but dying in any other sense, is failure. You may encounter characters that require items, methodical puzzle levels, intensely difficult platforming, tough levels that test your reaction time, and even bosses. Jump Off The Bridge is a meticulously crafted world that avoids the overuse of previously introduced sprites and characters. Every level feels unique, and is memorable.",
+    "desc_md": null,
+    "inst": "Purchase the game from steam: https://store.steampowered.com/app/1057780/Jump_Off_The_Bridge/\nDownload the game and from your Steam library,  right click on Jump Off The Bridge >  Manage > Browse local files. Copy the data.win, audiogroup2.dat and audiogroup3.dat from the assets folder to /ports/jumpoffthebridge/gamedata",
```

Comment:
In the future try to use ther inst_md for urls and such.
These will be shown on the website with proper links.

Reply:
okies will do so with the next batch of PRs :p